filesystem.spec

```diff
@@ -49,10 +53,6 @@ mkdir -p afs boot dev \
 #ln -snf etc/sysconfig etc/default
 ln -snf ../var/tmp usr/tmp
 ln -snf spool/mail var/mail
-ln -snf usr/bin bin
-ln -snf usr/sbin sbin
-ln -snf usr/lib lib
-ln -snf usr/%{_lib} %{_lib}
```

Comment:
Can you comment those out instead of removing them and add a fix me? It would be useful to keep in mind for usrmove.

Reply:
you should see all our changes from this 69091524bb8589efe2d129520b431fd14df26c51 commit

Reply:
updated to commit 404a52e1ed4ff56cd4e7aededdc810bbd0e43c62